tests/e2e/actor_features/actor_features_test.go

```diff
@@ -538,6 +538,50 @@ func TestActorFeatures(t *testing.T) {
 		require.NoError(t, err, "failed to reset reminder")
 	})
 
+	t.Run("Actor reminder delete self.", func(t *testing.T) {
+		// Each test needs to have a different actorID
+		actorID := "1001e"
+
+		// Reset reminder
+		res, err = httpDelete(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName))
+		if err != nil {
+			log.Printf("failed to reset reminder. Error='%v' Response='%s'", err, string(res))
+		}
+		require.NoError(t, err, "failed to reset reminder")
+
+		// Set reminder
+		req := actorReminderOrTimer{
+			Data:    "reminderdata",
+			DueTime: "1s",
+			Period:  "1s",
+		}
+		var reqBody []byte
+		reqBody, err = json.Marshal(req)
+		require.NoError(t, err, "failed to marshal JSON")
+		res, err = httpPost(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName), reqBody)
+		if err != nil {
+			log.Printf("failed to set reminder. Error='%v' Response='%s'", err, string(res))
+		}
+		require.NoError(t, err, "failed to set reminder")
+
+		time.Sleep(secondsToCheckTimerAndReminderResult * time.Second)
+
+		// Reset reminder
+		res, err = httpDelete(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName))
+		if err != nil {
+			log.Printf("failed to reset reminder. Error='%v' Response='%s'", err, string(res))
+		}
+		require.NoError(t, err, "failed to reset reminder")
```

Comment:
Shouldn't we try and get the reminder to make sure it's not there instead of deleting it again?